include/triton/Dialect/TritonGPU/IR/LinearLayoutConversions.h

```diff
@@ -26,7 +26,7 @@ namespace mlir::triton::gpu {
 //
 // All layouts have the following output dimensions.
 //
-//  "dimi" for i in 0..n-1: the location in the n'th logical dimension of the
+//  "dim[i]" for i in 0..n-1: the location in the n'th logical dimension of the
```

Comment:
By `dimi for i in 0..n-1` I meant the strings "dim0", "dim1", "dim2", ..., not "dim[0]" or the string at `dim[0]` for some array/map called `dim`.

Reply:
Got you